projects/canopy/src/lib/pipes/camel-case/camel-case.notes.ts

```diff
@@ -8,6 +8,8 @@ This pipe allows for a string to be transformed into camel case.
 Import the pipe in your module:
 
 ~~~
+import { LgCamelCasePipeModule } from '@legal-and-general/canopy';
```

Comment:
We never write the import in the documentation as it's usually automatically added by the IDE in your codebase so for consistency I would keep the notes as they were before.

Reply:
it wasn't obvious how to import the pipe, that's how I found the bug